tests/conftest.py

```diff
@@ -69,9 +70,26 @@ def partner_prod_api_key():
     easypost.api_key = default_key
 
 
+def check_expired_cassettes(expiration_days: int = 180):
+    """Checks for expired cassettes and throws errors if they are too old and must be re-recorded."""
+    test_name = os.environ.get("PYTEST_CURRENT_TEST").split(":")[-1].split(" ")[0]  # type: ignore
+    cassette_filepath = os.path.join("tests", "cassettes", f"{test_name}.yaml")
+
+    if os.path.exists(cassette_filepath):
+        expiration_age = datetime.datetime.now() - datetime.timedelta(days=expiration_days)
+        cassette_age = datetime.datetime.fromtimestamp(os.stat(cassette_filepath).st_mtime)
```

Comment:
Question: Is there no way to store a timestamp inside the file rather than relying on the file metadata? Moving the file would change the timestamp, even though the contents aren't changed.

Reply:
I checked the docs for `vcrpy` and there's no mention of a timestamp sadly. They don't include one we could use either.